apps/notifier/src/views/base.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>prueba generado</title>
+</head>
+<body>
+    <h1>si esto funciona, hoy me bebo la botella de vodka{{user_email}} </h1>
```

Comment:
I would also like to drink Vodka with you, but please, not inviting me through an email message 😞 😂